sysutils/vtm/Portfile

```diff
@@ -45,10 +45,6 @@ if {[variant_isset debug]} {
 
 configure.args-append   ${worksrcpath}/src
 
-variant mc description {Include GNU Midnight Commander with shell} {
-    depends_run-append  port:mc
-}
-
```

Comment:
why remove this variant?

Reply:
Now the startup behavior when opening a terminal in a shell is configurable and this variant does not make sense.